tests/UnifiedSpecTests/Util.php

```diff
@@ -33,6 +34,19 @@ public static function assertHasOnlyKeys($arrayOrObject, array $keys): void
         assertEmpty($diff, 'Unsupported keys: ' . implode(',', array_keys($diff)));
     }
 
+    /**
+     * @param string $executingObjectName
+     * @param string $operation
+     * @param array  $args
+     * @return void
+     */
```

Comment:
This can be removed. I only added it in the [PHPLIB-662 comment](https://jira.mongodb.org/browse/PHPLIB-662?focusedCommentId=4405627&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-4405627) to explain the API a bit more.

```suggestion
```